packages/stylelint-plugin-ckeditor5-rules/tests/rules/license-header.test.js

```diff
@@ -0,0 +1,298 @@
+/**
+ * @license Copyright (c) 2003-2022, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const { getTestRule } = require( 'jest-preset-stylelint' );
```

Comment:
Is it needed to have the `preset` object in `package.json`? Something is redundant here.